src/utils/src/ExponentialBackoffUtils.c

```diff
@@ -0,0 +1,223 @@
+#include "Include_i.h"
+
+//#define LOG_CLASS "XXXXX"
+
+STATUS getDefaultExponentialBackOffConfig(PExponentialBackoffConfig* ppPExponentialBackoffConfig) {
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    PExponentialBackoffConfig pExponentialBackoffConfig = NULL;
+
+    CHK(ppPExponentialBackoffConfig != NULL, STATUS_NULL_ARG);
+
+    pExponentialBackoffConfig = (PExponentialBackoffConfig) MEMCALLOC(0x00, SIZEOF(ExponentialBackoffConfig));
+    CHK(pExponentialBackoffConfig != NULL, STATUS_NOT_ENOUGH_MEMORY);
+
+    pExponentialBackoffConfig->maxRetryCount = KVS_INFINITE_EXPONENTIAL_RETRIES;
+    pExponentialBackoffConfig->maxWaitTime = HUNDREDS_OF_NANOS_IN_A_MILLISECOND * DEFAULT_KVS_MAX_WAIT_TIME_MILLISECONDS;
+    pExponentialBackoffConfig->retryFactorTime = HUNDREDS_OF_NANOS_IN_A_MILLISECOND * DEFAULT_KVS_RETRY_TIME_FACTOR_MILLISECONDS;
+    pExponentialBackoffConfig->minTimeToResetRetryState = HUNDREDS_OF_NANOS_IN_A_MILLISECOND * DEFAULT_KVS_MIN_TIME_TO_RESET_RETRY_STATE_MILLISECONDS;
+    pExponentialBackoffConfig->jitterFactor = DEFAULT_KVS_JITTER_FACTOR_MILLISECONDS;
+
+CleanUp:
+    (*ppPExponentialBackoffConfig) = pExponentialBackoffConfig;
+    LEAVES();
+    return retStatus;
+}
+
+STATUS resetRetryState(PExponentialBackoffState pExponentialBackoffState) {
```

Comment:
Consider naming of this and all other APIs to be a little more specific to avoid collisions

Reply:
Changed to more verbose name - `resetExponentialBackoffRetryState`
This hopefully will not collide.